ChangeLog

```diff
@@ -1,3 +1,25 @@
+XXX Jul XX 2025 Łukasz Stolarczuk <lukasz.stolarczuk@intel.com>
+
+	* Version 1.0.0-rc1
+
+	This is the first release with the backward compatibility guarantee.
```

Comment:
In 1.0.0 we should include here  broader explanation what UMF exactly is. 

Also please indicate that some api is experimental, and it not stable.